contracts/Tokens/VAI/VAIControllerStorage.sol

```diff
@@ -84,3 +84,11 @@ contract VAIControllerStorageG2 is VAIControllerStorageG1 {
     /// @notice Access control manager address
     address public accessControl;
 }
+
+contract VAIControllerStorageG3 is VAIControllerStorageG2 {
+    /// @notice The address of the prime contract
+    address public prime;
+
+    /// @notice Tracks if minting is enabled only for prime token holders
```

Comment:
I would specify that it can be `true` only if `prime` is not the zero address (assuming the rest of my comments)